pkg/streamingpromql/operators/functions/rate_increase.go

```diff
@@ -245,7 +258,6 @@ func calculateFloatRate(isCounter, isRate bool, rangeStart, rangeEnd int64, rang
 	averageDurationBetweenSamples := sampledInterval / float64(count-1)
 
 	extrapolationThreshold := averageDurationBetweenSamples * 1.1
-	extrapolateToInterval := sampledInterval
```

Comment:
Changes in calculateFloatRate here just to keep this both funcs in sync with changes from https://github.com/prometheus/prometheus/pull/16828